README.md

```diff
@@ -37,7 +37,29 @@ equivalent since the same input data can be used._
 
 ## Migration Guide
 
-Aqua has been deprecated. TBD - Link to Migration guide or more details here.
+Aqua has been deprecated:
+
+| Old | New | Library |
+| :---: | :---: | :---: |
+| qiskit.aqua.algorithms.amplitude_amplifiers | qiskit.algorithms.amplitude_amplifiers | qiskit-terra |
+| qiskit.aqua.algorithms.amplitude_estimators | qiskit.algorithms.amplitude_estimators | qiskit-terra |
+| qiskit.aqua.algorithms.classifiers | qiskit_machine_learning.algorithms.classifiers | qiskit-machine-learning |
+| qiskit.aqua.algorithms.distribution_learners | qiskit_machine_learning.algorithms.distribution_learners | qiskit-machine-learning |
+| qiskit.aqua.algorithms.eigen_solvers | qiskit.algorithms.eigen_solvers | qiskit-terra |
+| qiskit.aqua.algorithms.factorizers | qiskit.algorithms.factorizers | qiskit-terra |
+| qiskit.aqua.algorithms.minimum_eigen_solvers | qiskit.algorithms.minimum_eigen_solvers | qiskit-terra|
+| qiskit.aqua.algorithms.VQAlgorithm | qiskit.algorithms.VariationalAlgorithm | qiskit-terra |
+| qiskit.aqua.aqua_globals | qiskit.utils.aqua_globals | qiskit-terra|
+| qiskit.aqua.components.multiclass_extensions | qiskit_machine_learning | qiskit-machine-learning |
```

Comment:
There are no multiclass extensions any more. Given that the classifiers are based on sklearn it would be expected that users use the multiclass classifiers that slklearn provides.